ts/utils/convertDateToDistance.ts

```diff
@@ -0,0 +1,20 @@
+import { differenceInCalendarDays, format } from "date-fns";
+import I18n from "../i18n";
+
+// This function convert the distance from now to date in : H.mm, yesterday, D/MM/YY and DD/MM
+export function convertDateToDistance(date: string): string {
```

Comment:
* this function should get a `Date`, not a `string`
* better name it `convertDateToWordDistance` to make it clear this returns a string
* use jsdoc comment
* add tests